usaspending_api/search/v2/views/spending_by_transaction_grouped.py

```diff
@@ -0,0 +1,165 @@
+import copy
+import logging
+
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from usaspending_api.common.api_versioning import api_transformations, API_TRANSFORM_FUNCTIONS
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.elasticsearch.search_wrappers import TransactionSearch
+from usaspending_api.common.exceptions import UnprocessableEntityException
+from usaspending_api.common.helpers.generic_helper import get_generic_filters_message
+from usaspending_api.common.query_with_filters import QueryWithFilters
+from usaspending_api.search.filters.elasticsearch.filter import QueryType
+from usaspending_api.common.validator.award_filter import (
+    AWARD_FILTER,
+    AWARD_FILTER_NO_RECIPIENT_ID,
+)
+from usaspending_api.common.validator.pagination import PAGINATION
+from usaspending_api.common.validator.tinyshield import TinyShield
+from usaspending_api.search.v2.elasticsearch_helper import get_number_of_unique_terms_for_transactions
+from usaspending_api.search.v2.es_sanitization import es_minimal_sanitize
+from elasticsearch_dsl import A
+
+logger = logging.getLogger(__name__)
+
+API_VERSION = settings.API_VERSION
+
+
+@api_transformations(api_version=API_VERSION, function_list=API_TRANSFORM_FUNCTIONS)
+class SpendingByTransactionGroupedVisualizationViewSet(APIView):
+    """
+    This route provides aggregated information about transactions grouped by their prime awards. Additionally, allows
+    the transactions to be filtered.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction_grouped.md"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        models = [
+            {
+                "name": "fields",
```

Comment:
I don't believe this should be in use anymore. If that is the case, can you remove it from the TinyShield model and also the API Contract?